global.css

```diff
@@ -0,0 +1,5 @@
+@font-face {
```

Comment:
drop these changes, instead update the existing web/styles/global.css to fetch the fonts previously fetched from S3 to use world-id-assets.com

Reply:
drop changes to this file

Reply:
remove changes to this file